superset-frontend/packages/superset-ui-switchboard/src/switchboard.ts

```diff
@@ -90,7 +90,7 @@ function isError(message: Message): message is ErrorMessage {
 export class Switchboard {
   port: MessagePort;
 
-  name: string;
+  name = '';
```

Comment:
+1 to Ville's comment, I'd prefer `string` as well ... and rest of the file does it that way too!

Reply:
This is added because there can be instances of logging the error even when Switchboard is not initialised, which will lead to logging undefined if this change is not made.

Reply:
So, it should be `name: string = '';` then?

Reply:
That's automatically detected in typescript based on the value assigned.

Reply:
@sinhashubham95 makes sense, I hadn't thought of that 👍